nautobot/docs/user-guides/git-data-source.md

```diff
@@ -0,0 +1,257 @@
+# Git as a Data Source
+
+Using Git&trade; as a Data Source feature was developed to provide the ability to populate existing data, templates, scripts, and much more into Nautobot; while leveraging the benefits that GitHub already provides including issue tracking, discussions, pipelines, and approvals.
+
+For more technical details on how to use this feature, please see the documentation on [Git Repositories](https://nautobot.readthedocs.io/en/latest/models/extras/gitrepository/).
```

Comment:
Relative rather than absolute link please.
```suggestion
For more technical details on how to use this feature, please see the documentation on [Git Repositories](../models/extras/gitrepository.md).
```